### ISSUE TITLE
A Way To Say "Thank You!"

### ISSUE THREAD

**Issue by idfubar** [silviapfeiffer/silviapfeiffer.github.io#2]
Simple is definitely minimal... but also elegant (and, in this particular case, just what was needed); as such, is it O.K. to ask for a link to say "Thank You!" (e.g. PayPal, Gratipay, etc.)...?

**Comment by silviapfeiffer** [comment]
I'm a bit confused: are you intending to make a financial contribution?

**Comment by idfubar** [comment]
Yes, if it would help to say "thanks"... (but also just suggesting the idea in the event it wasn't considered when the page was created - or hasn't been fully explored)

= )

**Comment by silviapfeiffer** [comment]
Thanks for the idea! 🤔  Well keep it as a free community service. Please donate to a good cause in accessibility!
